src/core/services/utils/dom.ts

```diff
@@ -1805,7 +1805,11 @@ export class CoreDomUtilsProvider {
      */
     async openPopover<T = void>(options: OpenPopoverOptions): Promise<T | undefined> {
 
+        options = Object.assign({
+            showBackdrop: false,
+        }, options);
```

Comment:
```suggestion
        options = {
            showBackdrop: false,
            ...options,
        };
```
I prefer this because `Object.assign` returns type `any`, while the spread operator returns the right type.

Reply:
Code removed, it will be solved using css